src/PyCall.jl

```diff
@@ -366,8 +369,7 @@ hasproperty(o::PyObject, s::AbstractString) = pyhasproperty(o, s)
 
 #########################################################################
 
-keys(o::PyObject) = Symbol[m[1] for m in pycall(inspect."getmembers",
-                                PyVector{Tuple{Symbol,PyObject}}, o)]
+keys(o::PyObject) = convert(Vector{Symbol}, ccall((@pysym :PyObject_Dir), PyObject, (PyPtr,), o))
```

Comment:
We should really change this method since `o[...]` is no longer used for accessing members.  Probably:

```jl
@deprecate keys(o::PyObject) propertynames(o)
```